packages/core/src/constants/type/Config.ts

```diff
@@ -38,25 +45,33 @@ export type FullConfig = {
    */
   multicallVersion: 1 | 2
   /**
+   * When set to `true` improves performance by using own written encoders and decoders for multicall data.
    * @experimental
+   * @defaultValue false
    */
   fastMulticallEncoding?: boolean
   /**
+   * Don't listen on `disconnect` event from Metamask. Might be useful in dealing with [this Metamask bug](https://github.com/MetaMask/metamask-extension/issues/13375).
    * @experimental
+   * @defaultValue false
    */
   noMetamaskDeactivate?: boolean
   /**
+   * List of intended supported chains. If a user tries to connect to an unsupported chain an error value will be returned by useEthers.
    * @deprecated
+   * @defaultValue [ChainId.Mainnet, ChainId.Goerli, ChainId.Kovan, ChainId.Rinkeby, ChainId.Ropsten, ChainId.xDai]
```

Comment:
I know you copied it from current docs, but current docs are wrong - default value is `undefined`